examples/ocean_wind_mixing_and_convection.jl

```diff
@@ -28,62 +28,50 @@ using JLD2
 using Oceananigans
 using Oceananigans.Units: minute, minutes, hour
 
-# ## A vertically-stretched grid
+# ## The grid
 #
-# We use 32³ grid points with 2 meter grid spacing in the horizontal and
+# We use 32³ grid points with 2 m grid spacing in the horizontal and
 # varying spacing in the vertical, with higher resolution closer to the
-# surface. We use a two-parameter generating function to specify the
-# vertical cell interfaces:
+# surface,
 
-Nz = 24 # number of points in the vertical direction
-Lz = 32 # domain depth
-refinement = 1.2 # controls spacing near surface (higher means finer spaced)
-stretching = 8   # controls rate of stretching at bottom 
+σ = 1.1 # stretching factor
+Nz = 24
+Lz = 32
 
-## Normalized height ranging from 0 to 1
-h(k) = (k - 1) / Nz
-
-## Linear near-surface generator
-ζ₀(k) = 1 + (h(k) - 1) / refinement
-
-## Bottom-intensified stretching function 
-Σ(k) = (1 - exp(-stretching * h(k))) / (1 - exp(-stretching))
-
-## Generating function
-z_faces(k) = Lz * (ζ₀(k) * Σ(k) - 1)
+hyperbolically_spaced_faces(k) = - Lz * (1 - tanh(σ * (k - 1) / Nz) / tanh(σ))
 
 grid = VerticallyStretchedRectilinearGrid(size = (32, 32, Nz), 
                                           x = (0, 64),
                                           y = (0, 64),
-                                          z_faces = z_faces)
+                                          z_faces = hyperbolically_spaced_faces)
 
 # We plot vertical spacing versus depth to inspect the prescribed grid stretching:
 
 plot(grid.Δzᵃᵃᶜ[1:Nz], grid.zᵃᵃᶜ[1:Nz],
-     marker = :circle,
-     ylabel = "Depth (m)",
-     xlabel = "Vertical spacing (m)",
-     legend = nothing)
+      marker = :circle,
+      ylabel = "Depth (m)",
+      xlabel = "Vertical spacing (m)",
+      legend = nothing)
 
 # ## Buoyancy that depends on temperature and salinity
 #
 # We use the `SeawaterBuoyancy` model with a linear equation of state,
 
 buoyancy = SeawaterBuoyancy(equation_of_state=LinearEquationOfState(α=2e-4, β=8e-4))
 
-# where ``α`` and ``β`` are the thermal expansion and haline contraction
+# where $α$ and $β$ are the thermal expansion and haline contraction
```

Comment:
```suggestion
# where ``\alpha`` and ``\beta`` are the thermal expansion and haline contraction
```

Reply:
I believe the letters still render properly if we use them literally rather than using latex commands. That's definitely preferred because we can read the text easier from the script, right? Gotta be careful not to orient the examples too much toward markdown, because they are meant to be code examples, too...

Reply:
Or is this no longer true? I thought it was but I can't find it documented.

Reply:
Two remarks:

1. The letters render differently. I prefer the way ``\alpha`` looks compared to ``α``, but that's just a personal preference.
2. Either way, ` `` ... `` ` is the way to LaTeX math inline, not `$ ...$`.

Reply:
Yeah, I see what you're saying. I don't read the markdown much --- I usually read the code. So my priorities are different. Why do they render differently though? Shouldn't they be the same?

Reply:
It's the difference between

```
B(y, z) = f ∂_z ψ + N² z
```

and

```
B(y, z) = f \partial_z \psi + N^2 z
```

Reply:
I don't know why they render differently... some MathJax thing....

It may have been me who pushed changes in `ocean_wind_mixing_and_convection.jl` in this PR? makes no sense... let's just remove any changes in this PR, no?

Reply:
What do you mean "your priorities are different"?

Reply:
The difference only appears in the rendered docs.

Compare, e.g., the χ, κ, and ζ between:

A. with ``κ``, ``ζ``, ``χ``)
https://clima.github.io/OceananigansDocumentation/previews/PR1847/generated/horizontal_convection/#Output

and
Β. with ``\kappa``, ``\zeta``, ``\chi``)
https://clima.github.io/OceananigansDocumentation/dev/generated/horizontal_convection/#Output
https://clima.github.io/OceananigansDocumentation/dev/generated/horizontal_convection/#Load-saved-output,-process,-visualize

Which do you like better?

Reply:
Yeah, I totally agree that the way ``ζ`` renders is completely ugly. Is it the difference between a font for Greek letters and a "math" font?

We could open an issue in Documenter and ask that ``ζ`` get rendered as ``\zeta``. Not sure if that's possible but it'd be nice.

Reply:
> I don't know why they render differently... some MathJax thing....
> 
> It may have been me who pushed changes in `ocean_wind_mixing_and_convection.jl` in this PR? makes no sense... let's just remove any changes in this PR, no?

I'm not stressed about it if they are positive changes.

Reply:
> Yeah, I totally agree that the way ζ renders is utterly ugly. Is it the difference between a font for Greek letters and a "math" font?

``ζ`` renders like Times New Roman ζ..., ``\zeta`` renders like Computer Modern ζ. We could open a Documenter issue, but I think it's MathJax-related. I have the slight impression (but I may be wrong) that with KaTeX this was not an issue.

Reply:
I'm relieved that you find the former version of the zetas above "utterly ugly". At least our aesthetics align.

Reply:
Huh. But can't Documenter just replace `ζ` with `\zeta` prior to compilation with MathJax if it wanted? Or maybe this would have to be a julia markdown rule or something. Hmm.